src/main.cpp

```diff
@@ -310,6 +314,7 @@ void getConfigSchemaJson(JsonVariant json)
   JsonObject fadeIntervalUs = properties.createNestedObject("fadeIntervalUs");
   fadeIntervalUs["type"] = "integer";
   fadeIntervalUs["minimum"] = 0;
+  fadeIntervalUs["description"] = "Global variable applied to all strips - Time to fade from off to on, or vice versa, in milliseconds (default is 500ms)";
```

Comment:
"Default time to fade from off -> on (and vice versa), in microseconds (defaults to 500us)"